src/vmm/src/guest_config/mod.rs

```diff
@@ -0,0 +1,5 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#[cfg(cpuid)]
+pub mod cpuid;
```

Comment:
Isn't locking cpuid module behind the config flag creates more issues? If config flag is not present, the code in `vcpu/x86_64/mod.rs` will fail to compile, because it relies on `guest_config::couid`, but is not locked behind the cfg flag.

Reply:
This was configuration for the existing module declaration. There may be a case for generalizing this to x86_64 configuration, but outside of scope for now.